include/Editor.h

```diff
@@ -80,7 +80,7 @@ private slots:
 	///
 	/// \param	record	If set true, the editor's toolbar will contain record
 	///					buttons in addition to the play and stop buttons.
-	Editor(bool record = false, bool record_step = false);
+	Editor(bool record = false, bool recordAccompany = false, bool record_step = false);
```

Comment:
```suggestion
	Editor(bool record = false, bool recordAccompany = false, bool record_step = false);
```
Better naming? Maybe `allowRecording`? or `isRecordingEnabled`?

Reply:
@regulus79 and I discussed this previously. These variables are from the constructor of the editor, each name is assigned to one of the recording buttons. Do you think it is really neccesary to change the names of the variables to

Editor(bool allowRecording = false, bool allowRecordingAccompany = false, bool allowStepRecording = false);

or

Editor(bool isRecordingEnabled = false, bool isRecordingAccompanyEnabled = false, bool isStepRecordingEnabled = false);

?
Personally, I think it is unnecessary.

Reply:
It's only true in the piano roll and song editor; the automation editor and pattern editor default to false because they don't support the same kind of recording.